README.md

```diff
@@ -51,13 +51,13 @@ as they may have been used for past encryption operations.
   * Supports Additional Authenticated Data (AAD) for all KMSes except Vault Transit.
 
 A
-[`multiwrapper`](https://github.com/hashicorp/go-kms-wrapping/tree/master/wrappers/multiwrapper)
+[`multiwrapper`](https://github.com/hashicorp/go-kms-wrapping/tree/v2/wrappers/multiwrapper)
```

Comment:
```suggestion
[`multiwrapper`](https://github.com/hashicorp/go-kms-wrapping/tree/v2/multiwrapper)
```

Should it be this instead since we're moving it up?

Reply:
Thanks!